crates/oxc_transformer/src/react_jsx/options.rs

```diff
@@ -7,8 +7,7 @@ use serde::Deserialize;
 #[serde(rename_all = "camelCase")]
 pub struct ReactJsxOptions {
     /// Decides which runtime to use.
-    #[serde(default)]
-    pub runtime: ReactJsxRuntime,
+    pub runtime: Option<String>,
```

Comment:
We can't use a specific enum type here, because if deserialized fails, `serde` will fallback to the default value, the runtime/invalid-runtime/input.js test case requires emitting error when runtime neither `automatic` nor `classic`

Reply:
`ReactJsxOptions` is user facing, we shouldn't provide vague values such as `Option<String>`, we should implement the `std::str::FromStr` trait instead.